x-pack/plugins/infra/public/components/asset_details/utils.ts

```diff
@@ -21,3 +23,23 @@ export const getDefaultDateRange = () => {
     to: new Date(now).toISOString(),
   };
 };
+
+export enum INTEGRATION_NAME {
+  nginx = 'nginx',
+  kubernetes = 'kubernetes',
+}
+export const INTEGRATIONS = {
+  [INTEGRATION_NAME.nginx]: ['nginx.stubstatus', 'nginx.access'],
+  [INTEGRATION_NAME.kubernetes]: ['kubernetes.node'],
+};
+
+export const getIntegrationAvailable = (
+  integration: INTEGRATION_NAME,
+  metadata?: InfraMetadata | null
+) => {
+  if (metadata) {
+    return metadata?.features?.some((f) => INTEGRATIONS[integration].includes(f.name))
+      ? integration
+      : null;
+  }
```

Comment:
Good idea, thanks for the suggestion! I was thinking about having the option of which integration to pick but it's not really needed in this case because we will need all of them.